google/datalab/contrib/pipeline/airflow/airflow_vm_create.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+VM_NAME="instance-7"
```

Comment:
These should be options/flags.

Reply:
Well, I'm actually writing this script to be a set of commands for some easy testing which I can easily copy-paste into a notebook. Hence I've not used flags. It's not actually meant for being executed and I will delete it once the notebook is ready.

Reply:
We can just give alpha users a set of .sh files. So that way, what you use for testing is what we give out.

Reply:
Hmm ... possibly. I have a slight preference for giving out a notebook though.